src/scripts/Joyride.jsx

```diff
@@ -97,6 +103,12 @@ class Joyride extends React.Component {
     type: 'single'
   };
 
+  componentWillMount() {
```

Comment:
I've realized this is the wrong lifecycle hook, because it's possible the rest of the DOM is not mounted either, so there won't be targets.  I'll be pushing a change shortly.